aws/resource_aws_ecs_service.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/hashicorp/terraform/helper/validation"
 )
 
+const schedulingStrategyDaemon = "DAEMON"
```

Comment:
You should just be able to use `ecs.SchedulingStrategyDaemon` (and its equivalent `ecs.SchedulingStrategyReplica`) here instead.